packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -74,6 +74,7 @@ export const comments: { readonly [key in TokenName]: string } = {
     spinnerSmallHeight: 'Small height (16px) for a spinner component',
     spinnerMediumHeight: 'Medium height (32px) for a spinner component',
     spinnerLargeHeight: 'Large height (64px) for a spinner component',
+    tableFitRowsHeight: 'The height of the table when all rows are visible',
```

Comment:
nit: Since the common case for most of our tokens is to have the same value across an entire application, it could be helpful to highlight that this one is different.
```suggestion
    tableFitRowsHeight: 'The height of the table when all rows are visible. It is set automatically to the correct value within the scope of each table.',
```